ts/Core/Axis/Color/ColorAxis.ts

```diff
@@ -564,21 +565,38 @@ class ColorAxis extends Axis implements AxisLike {
         this.setLegendColor();
 
         // Create the gradient
-        item.legendSymbol = this.chart.renderer.rect(
-            0,
-            (legend.baseline as any) - 11,
-            width,
-            height
-        ).attr({
-            zIndex: 1
-        }).add(item.legendGroup);
-
+        if (!item.legendSymbol) {
+            item.legendSymbol = this.chart.renderer.rect(
+                0,
+                (legend.baseline as any) - 11,
```

Comment:
We could (try to) reduce `as any` in this file while the code is modified.
Here `(legend.baseline || 0)` could be used.